settings.gradle.kts

```diff
@@ -1,10 +1,16 @@
 enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
 
 pluginManagement {
+  val dokkaVersion: String by settings
+
   repositories {
     mavenCentral()
     gradlePluginPortal()
   }
+
+  plugins {
+    id("org.jetbrains.dokka") version dokkaVersion
+  }
```

Comment:
Question: why is the Dokka plugin specified in the settings file?

Reply:
This is just because it is the only mechanism (as far as I know) to specify the version of a plugin using a value coming from gradle.properties

Reply:
Don't use `gradle.properties` if you can avoid it entirely.  
Use the version catalog (`gradle/libs.versions.toml`) and `alias`ing, and import/activate the plugin explicitly on each project.

Reply:
Ok, I am moving the dokka version to the toml file and refactor settings.gradle.kts accordingly